src/concrete/ml/torch/compile.py

```diff
@@ -110,7 +110,7 @@ def build_quantized_module(
     # bit-width, but for tracing we only take a single one. We need the ONNX tracing batch size to
     # match the batch size during FHE inference which can only be 1 for the moment.
     dummy_input_for_tracing = tuple(
-        torch.from_numpy(val[[0], ::]).float() for val in inputset_as_numpy_tuple
+        torch.from_numpy(val[[0], ::]) for val in inputset_as_numpy_tuple
     )
```

Comment:
That change is breaking lots of tests. Not sure why this .float() is so important.

Reply:
what was the issue ?

Reply:
We need the data to be float32 here if there are float64 somehow. Otherwise torch isn't happy during tracing in the onnx dump.

Reply:
worth adding a comment about it right above so we remember it then !

Reply:
also, why do you need to pass integers now (that's what you are trying to do right ?)

Reply:
Because embedding layer want integers / labels.

Reply:
ok I guess it makes sense, well as said above I would at least add a comment on that + the float thing then !

Reply:
Added a comment

Reply:
thanks ! just, unless you re not sure about it, could you instead write why it fails with fp64 and why we need integer support (more than just say that it fails without it) ? it's just that I feel we'll stumble again on that code one day and we'll not be sure anymore why it's needed

Reply:
The error was a mismatch of between to matrix for the matmul. I didn't investigate as it was going down the onnx library before failing.